src/components/QRDialog.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="qrDialog__content">
+		<h2>{{ title }}</h2>
+		<div>
+			<img :src="uri"
+				:title="text"
+				:alt="t('forms', 'QR code representation of {title}', { title: title })">
```

Comment:
Just another quick fix: the QR code doesn't represent the form title, but the share link. So we need to use `text` here, too:
```suggestion
				:alt="t('forms', 'QR code representation of {text}', { text: text })">
```